### ISSUE TITLE
amp-ima-video in amp-carousel: video did not pause on swipe

### ISSUE THREAD

**Issue by dzuendorf** [ampproject/amphtml#23365]
The amp-ima-video did not pause on swipe/ slide change in an amp-carousel. The amp-video works fine.

Here is an code example testet in AMP Playground:

```
<amp-carousel height="300" layout="fixed-height" type="slides">
    <amp-video width="480" height="270"
  		src="/static/samples/video/tokyo.mp4"
  		poster="/static/samples/img/tokyo.jpg"
  		layout="responsive"
  		controls>
  		<source type="video/mp4"
    		src="/static/samples/video/tokyo.mp4">
	</amp-video>
    <amp-ima-video width="640" height="360" layout="responsive"
    	data-tag="https://pubads.g.doubleclick.net/gampad/ads?sz=640x480&iu=/124319096/external/ad_rule_samples&ciu_szs=300x250&ad_rule=1&impl=s&gdfp_req=1&env=vp&output=vmap&unviewed_position_start=1&cust_params=deployment%3Ddevsite%26sample_ar%3Dpremidpost&cmsid=496&vid=short_onecue&correlator="
    	data-poster="/static/samples/img/ima-poster.jpg">
    	<source src="https://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4" type="video/mp4">
  	</amp-ima-video>
    <amp-img src="/static/samples/img/image1.jpg" width="400" height="300" alt="a sample image"></amp-img>
    <amp-img src="/static/samples/img/image2.jpg" width="400" height="300" alt="another sample image"></amp-img>
    <amp-img src="/static/samples/img/image3.jpg" width="400" height="300" alt="and another sample image"></amp-img>
</amp-carousel>
```

## What browsers are affected?

Testet in Chrome and Safari.

**Comment by micajuine-ho** [comment]
Hi @dzuendorf, thank you for your issue report.

Your example actually demonstrates two issues:

1. The video controls (play/pause) are not visible on the ima-video 
2. When the controls are visible (possible by using a different video), the video still didn't pause

The first issue should be fixed by #23660, and I have fixed the second issue in the PR above.
